meta-ostro/recipes-swupd/swupd-client/swupd-client_%.bbappend

```diff
@@ -36,3 +37,12 @@ do_install_append () {
     # Don't install and enable check-update.timer by default
     rm -f ${D}/${systemd_system_unitdir}/check-update.* ${D}/${systemd_system_unitdir}/multi-user.target.wants/check-update.*
 }
+
+pkg_postinst_${PN}_append () {
+    # Setting a label explicitly on the directory prevents it
+    # from inheriting other undesired attributes like security.SMACK64TRANSMUTE
+    # from upper folders (see xattr-images.bbclass for details).
+    if [ "${DISTRO_FEATURES#*smack}" != "${DISTRO_FEATURES}" ]; then
```

Comment:
This check is not the usual way checking it. Is ${ #* } supported by POSIX shell?

The more idiomatic way of doing it would be

    if ${@bb.utils.contains('DISTRO_FEATURES', 'smack', 'true', 'false', d}; then

This expands to "if true; then" when smack is enabled.

Patch looks fine otherwise.